pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.ibm.websphere.appserver.features</groupId>
     <artifactId>intoto4j</artifactId>
-    <version>2.0.0</version>
```

Comment:
If the `WarResourceDescriptor` class is being renamed to `FileResourcesDescriptor` - and thus eliminating the `WarResourceDescriptor` class - the version should be changed to `3.0.0` because the change is not backwards compatible. Anyone using this library and the `WarResourceDescriptor` class would be broken upgrading from 2.0.0. to 2.1.0.